owledge/db/seeds.rb

```diff
@@ -12,7 +12,13 @@
                                 In mathematics, graph theory is the study of graphs, which are mathematical structures
                                 used to model pairwise relations between objects.
                               }.join(" ")
-                            }])
+                            })
+sorting = Note.create({
+                            title: 'Sorting Algorithms',
+                            description: %w{
+                              In computer science, a sorting algorithm is an algorithm that puts elements of a list in a certain order.
```

Comment:
nit: on a one liner, you can probably use %q and drop the .join(" ") at the end. I added that for cases where we needed to wrap newlines.

Reply:
Im thinking of leaving this as is for now